.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,83 @@
+name: 🧪 Unit tests
+
+on:
+  workflow_call:
+
+jobs:
+  unit-tests:
+    name: Test on ${{matrix.runner}}
+    runs-on: ${{matrix.runner}}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - runner: windows-latest
+            godot-suffix: win64.exe
+          - runner: ubuntu-latest
+            godot-suffix: linux.x86_64
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          submodules: false # don't initialize submodules automatically
```

Comment:
Why the additional step instead of this?

Reply:
Can't specify which submodules to check out... Don't need `sentry-native` and such.

Reply:
All good